TouristEquipmentRental/User.cpp

```diff
@@ -2,9 +2,8 @@
 #include <algorithm>
 #include <string>
 #include <vector>
-
 #include "User.h"
-#include "UserType.h"
+//#include "UserType.h"
```

Comment:
What is the purpose of this comment?

Reply:
There is no such file as "UserType.h"

Reply:
Ok, so probably I may have deleted it and have forgotten to delete this include. Feel free to delete it